### PR TITLE
feature: support checking whether there are Chinese links on english …

### PR DESCRIPTION
**Pull request by liwang2019**
…pages and vice versa.

### CODE REVIEW

File: lib/traverse.js [hunk 1]
```diff
@@ -32,6 +33,14 @@ async function traverse(page, url) {
     return doNext(page);
   }
 
+  const text = await response.text();
+  const chineseCount =  text.match(/[\u4E00-\u9FA5]/g) ? text.match(/[\u4E00-\u9FA5]/g).length : 0;
+  const sourceLang = source.indexOf('/cn/') === -1 ? 'en' : 'cn';
+
+  if(((sourceLang === 'cn' && chineseCount <= 10) || (sourceLang === 'en' && chineseCount > 10)) && !(source.replace('/cn/', '/en/') === url || url.replace('/cn/', '/en/') === source)) {
```
Comment (meathill) [inline lib/traverse.js:40]: These conditions should be calculated into some variables, like this:

```js
const con1 = ....;
const con2 = ....;
const con3 = ....;
if (con1 && con2 || con3) {

}
```

**Review by meathill — CHANGES_REQUESTED** [review]

File: index.js [hunk 2]
```diff
@@ -58,6 +58,11 @@ const {startUrl, domains} = config;
     if (!isEmpty(broken)) {
       console.table(broken);
     }
+
+    if (langNotMatch && langNotMatch.length !== 0) {
```
[the remark this answers is not in the document]
Comment (liwang2019) [reply index.js:62]: changed.

File: lib/traverse.js (hunk 1 above)
Comment (liwang2019) [reply lib/traverse.js:40]: changed.

**Review by meathill — APPROVED** [review]
LGTM.

**Pull request merged**